README.md

```diff
@@ -142,6 +142,7 @@ Note that as part of the onboarding process your repository name may be normaliz
 [symbols-weakmap]: https://github.com/tc39/proposal-symbols-as-weakmap-keys
 [symbols-weakmap-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2022-06/jun-07.md#symbols-as-weakmap-keys
 [regexp-v-flag]: https://github.com/tc39/proposal-regexp-v-flag
+[regexp-v-flag-tests]: https://github.com/tc39/test262/pull/3523
```

Comment:
Those are just _some_ of the tests. Here’s the other 2 PRs: https://github.com/tc39/test262/issues/3496#issuecomment-1105628902

Reply:
It might be a tedious exercise to list _all_ of the test262 PRs for every proposal; maybe we should make the checkmarks link into the test262 tree instead?

Reply:
Tests are not always all in the same directory, as is the case here. Maybe the thing to do is to refer to the feature tag as used in the frontmatter for the tests, as registered in https://github.com/tc39/test262/blob/main/features.txt? For this particular feature that’d be `regexp-v-flag`.

Alternatively, we could just point to the Test262 tracking issue, or in this case, the issue comment I linked to in my earlier comment.

Reply:
I like the idea of using the feature flag

Reply:
Have a look at https://github.com/tc39/proposals/pull/432...